composer.json

```diff
@@ -1,82 +1,48 @@
-            "app/AppKernel.php",
-            "app/AppCache.php"
-        ]
+            "eLife\\": "./src"
+        }
     },
     "autoload-dev": {
         "psr-4": {
-            "Csa\\Bundle\\GuzzleBundle\\": "./vendor-extra/Csa/GuzzleBundle",
-            "spec\\eLife\\HypothesisClient\\": "./spec",
-            "tests\\eLife\\HypothesisClient\\": "./tests"
+            "tests\\eLife\\": "./tests"
         }
     },
     "require": {
-        "php": ">=7.0",
-        "csa/guzzle-bundle": "^2.2",
-        "guzzlehttp/promises": "^1.3",
-        "incenteev/composer-parameter-handler": "^2.1",
-        "psr/http-message": "^1.0",
-        "sensio/distribution-bundle": "^5.0",
-        "sensio/framework-extra-bundle": "^5.0",
-        "symfony/monolog-bundle": "^3.1",
-        "symfony/polyfill-apcu": "^1.5",
-        "symfony/serializer": "^3.3",
-        "symfony/symfony": "^3.3",
-        "twig/twig": "^2.4"
+        "php": "^7.0",
+        "beberlei/assert": "^2.7",
+        "doctrine/annotations": "1.4.0",
+        "doctrine/cache": "~v1.6.2",
+        "doctrine/instantiator": "~1.0.5",
+        "elife/api-problem": "^1.0",
+        "elife/api-sdk": "^1.0@dev",
+        "elife/bus-sdk": "dev-master",
+        "elife/ping": "^1.0",
+        "guzzlehttp/guzzle": "^6.3",
+        "knplabs/console-service-provider": "^2.1",
+        "silex/silex": "^2.2",
+        "symfony/property-access": "^3.3",
+        "symfony/serializer": "^3.3"
     },
     "require-dev": {
         "elife/proofreader-php": "dev-master",
-        "guzzlehttp/guzzle": "^6.3",
+        "giorgiosironi/eris": "^0.9.0",
         "mtdowling/jmespath.php": "^2.4",
-        "phpspec/phpspec": "^4.0",
-        "phpspec/prophecy": "^1.7",
         "phpunit/phpunit": "^5.7",
-        "sensio/generator-bundle": "^3.1",
-        "symfony/phpunit-bridge": "^3.3"
+        "silex/web-profiler": "^2.0",
+        "symfony/browser-kit": "^3.3"
+    },
+    "scripts": {
+        "server": "php -S 0.0.0.0:8088 -t web web/app_dev.php",
+        "server:prod": "php -S 0.0.0.0:8088 -t web web/app_prod.php"
     },
     "suggest": {
-        "guzzlehttp/guzzle": "^6.3, to use the Guzzle6HttpClient adapter",
         "mtdowling/jmespath.php": "^2.4, to use the Result::search() method"
     },
-    "scripts": {
-        "symfony-scripts": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
-        ],
-        "post-install-cmd": [
-            "@symfony-scripts"
-        ],
-        "post-update-cmd": [
-            "@symfony-scripts"
-        ]
-    },
+    "minimum-stability": "dev",
```

Comment:
Always a little suspicious in a project, what needs this?

Reply:
Years ago this was necessary to install private libraries which hadn't a defined release version, but only `master`. Perhaps `api-sdk` and such don't need it?

Reply:
this can go. :composer-noob: